to.etc.domui/src/main/java/to/etc/domui/converter/ConverterRegistry.java

```diff
@@ -65,6 +65,7 @@
 		register(new DomainListConverterFactory()); // Accepts anything having domain value list
 		register(new DateConverterFactory());
 		register(new LocalDateTimeConverterFactory());
+		register(new ZonedDateTimeConverterFactory());
```

Comment:
well, this got in by some side effect... that supposed to be already in skarp-prd?